docs/user-guide/ui.md

```diff
@@ -2,76 +2,68 @@
 assignees:
 - bryk
 - mikedanese
+- rf232
 
 ---
 
 
-Dashboard (the web-based user interface of Kubernetes) allows you to deploy containerized applications to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resources itself. You can use it for getting an overview of applications running on the cluster, as well as for creating or modifying individual Kubernetes resources and workloads, such as Daemon sets, Pet sets, Replica sets, Jobs, Replication controllers and corresponding Services, or Pods.
+Dashboard (the web-based user interface of Kubernetes) allows you to deploy containerized applications to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resources itself. You can use it for getting an overview of applications running on the cluster, as well as for creating or modifying individual Kubernetes resources and workloads, such as Daemon Sets, Stateful Sets, Replica Sets, Jobs and corresponding Services, or Pods.
 
-Dashboard also provides information on the state of Pods, Replication controllers, etc. and on any errors that might have occurred. You can inspect and manage the Kubernetes resources, as well as your deployed containerized applications. You can also change the number of replicated Pods, delete Pods, and deploy new applications using a deploy wizard.
+Dashboard also provides information on the state of Deployments, Replica Sets, etc. and on any errors that might have occurred. You can inspect and manage the Kubernetes resources and applications deployed on the cluster. For example, you can scale a Deployment, initiate a rolling update, restart a pod or deploy new applications using a deploy wizard.
 
-By default, Dashboard is installed as a cluster addon. It is enabled by default as of Kubernetes 1.2 clusters.
+![Kubernetes Dashboard UI](/images/docs/ui-dashboard.png)
 
 * TOC
 {:toc}
 
-## Dashboard access
+## Accessing Dashboard UI
 
-Navigate in your Browser to the following URL:
-```
-https://<kubernetes-master>/ui
-```
-This redirects to the following URL:
-```
-https://<kubernetes-master>/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
-```
-The Dashboard UI lives in the `kube-system` [namespace](/docs/admin/namespaces/), but shows all resources from all namespaces in your environment.
-
-If you find that you are not able to access Dashboard, you can install and open the latest stable release by running the following command:
+### Command line proxy
+The easiest way to access Dashboard is to use kubectl. Run the following command in your desktop environment:
```

Comment:
Remove "the easiest way."

"You can access Dashboard using the `kubectl` command-line tool by running the following command:"